utils/domoticz/Makefile

```diff
@@ -75,7 +76,7 @@ CMAKE_OPTIONS += \
 
 TARGET_CFLAGS += -flto
 TARGET_CXXFLAGS += -DWITH_GPIO -flto
-TARGET_LDFLAGS += -lpython3.9
+TARGET_LDFLAGS += -lpython$(PYTHON3_VERSION)
```

Comment:
Wrong. This just needs to be removed. This was a hack from before domoticz had proper python 3.9 support. It's not needed anymore.

Reply:
OK